packages/tizen_audio_manager/tizen/src/tizen_audio_manager_plugin.cc

```diff
@@ -10,73 +10,38 @@
 #include <flutter/method_channel.h>
 #include <flutter/plugin_registrar.h>
 #include <flutter/standard_method_codec.h>
-#include <sound_manager.h>
 
 #include <memory>
 #include <string>
 
+#include "audio_manager.h"
+#include "audio_manager_error.h"
 #include "log.h"
 
 namespace {
-
-constexpr std::array<std::pair<std::string_view, sound_type_e>, 8> kSoundTypes{
-    {{"system", SOUND_TYPE_SYSTEM},
-     {"notification", SOUND_TYPE_NOTIFICATION},
-     {"alarm", SOUND_TYPE_ALARM},
-     {"ringtone", SOUND_TYPE_RINGTONE},
-     {"media", SOUND_TYPE_MEDIA},
-     {"call", SOUND_TYPE_CALL},
-     {"voip", SOUND_TYPE_VOIP},
-     {"voice", SOUND_TYPE_VOICE}}};
-
-bool ConvertSoundTypeToString(sound_type_e type, std::string &str) {
-  auto iter = kSoundTypes.begin();
-
-  for (; iter != kSoundTypes.end(); iter++) {
-    if (iter->second == type) {
-      str = iter->first;
-      return true;
-    }
-  }
-
-  return false;
-}
-
-bool ConvertStringToSoundType(const std::string &str, sound_type_e &type) {
-  auto iter = kSoundTypes.begin();
-
-  for (; iter != kSoundTypes.end(); iter++) {
-    if (iter->first == str) {
-      type = iter->second;
-      return true;
-    }
-  }
-
-  return false;
-}
+typedef flutter::MethodCall<flutter::EncodableValue> FlMethodCall;
```

Comment:
Please add a blank line at the beginning of the namespace block.

Reply:
done